app/Http/Controllers/API/Profile/UploadAvatarController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Http\Controllers\API\Profile;
+
+use Brewmap\Contracts\StoreFile;
+use Brewmap\Http\Controllers\Controller;
+use Brewmap\Http\Requests\Profile\UpdateProfileAvatarRequest;
+use Illuminate\Http\JsonResponse;
+
+class UploadAvatarController extends Controller
+{
+    public function store(UpdateProfileAvatarRequest $request, StoreFile $service): JsonResponse
+    {
+        /** @var string $pathToAvatar */
```

Comment:
This is probably unnecessary as there is already return type in `StoreFile` interface:
https://github.com/brewmap/brewmap/blob/a683f75c7315f85f6ac951278f39adb4ffd08445/app/Contracts/StoreFile.php#L11
```suggestion
```